### DISCUSSION TITLE
How can we inject different environments and variables configured

### DISCUSSION THREAD

**Discussion by saeedsokar** [apideck-libraries/portman#486] [Q&A] [answered]
Hello,
How can I inject different variable types like Environment and collection variables in Portman? 
and How to inject different environments in Portman like to have set up a dev and stage environments.
in the generated Postman collection
Thanks.

**Comment by thim81** [comment, marked as answer]
Portman allows you to define .ENV files to load variables in Postman Or pass along a .env file by using the `--envFile`  parameter 
https://github.com/apideck-libraries/portman#environment-variables-as-postman-variables

Does that answer? Or did you have something else in mind?
